rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -541,6 +541,11 @@ def element_attribute(element, name)
           execute :getElementAttribute, id: element.ref, name: name
         end
 
+        # Backwards compatibility for w3c
+        def element_property(element, name)
+          execute :getElementAttribute, id: element, name: name
```

Comment:
@AutomatedTester - how is Python doing this? Are you going to support calling `getElementProperty` on a non-w3c driver?